compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -182,6 +182,7 @@
     UNINITIALIZED_VARIABLE("uninitialized.variable"),
     INVALID_ANY_VAR_DEF("invalid.any.var.def"),
     INVALID_RECORD_LITERAL("invalid.record.literal"),
+    DUPLICATE_RECORD_LITERAL("duplicate.record.literal"),
```

Comment:
Shall we rename this to something like `DUPLICATE_KEY_IN_RECORD_LITERAL` or something? `DUPLICATE_RECORD_LITERAL` conveys a different meaning

Reply:
good idea, will change.